go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -163,6 +163,10 @@ func commandCreateKeyspace(cmd *cobra.Command, args []string) error {
 
 	var snapshotTime *vttime.Time
 	if topodatapb.KeyspaceType(createKeyspaceOptions.KeyspaceType) == topodatapb.KeyspaceType_SNAPSHOT {
+		if createKeyspaceOptions.DurabilityPolicy != "none" {
```

Comment:
I wonder whether we should just log a warning and ignore the parameter instead of erroring out.

Reply:
I think that ignoring the durability policy could be a little non-intuitive from the user perspective. I think it is better to explicitly ask them to rerun the command without specifying the durability at all.

Reply:
I'm inclined toward the "error out" approach; the error message is clear and the user can just re-run the command with the correct incantation (vs maybe getting an unexpected "surprise" behavior).

Only thing I would change is `--durability-policy must not be specified ...` => `--durability-policy cannot be specified`

Reply:
sgtm